complementary_filter:
  ros__parameters:
    alpha: 0.93
    sensor_type: "marg"
    input_topic: "/marg/raw_data"
    output_topic: "/marg/complementary_filter"
    magnetometer_declination: 0.0
    magnetometer_negate_y: false